pkg/monitortests/node/legacynodemonitortests/kubelet.go

```diff
@@ -224,12 +224,20 @@ func testKubeAPIServerGracefulTermination(events monitorapi.Intervals) []*junita
 }
 
 func testContainerFailures(adminRestConfig *rest.Config, events monitorapi.Intervals) []*junitapi.JUnitTestCase {
-	containerExits := make(map[string][]string)
-	failures := []string{}
+	openshiftNamespaces := sets.Set[string]{}
```

Comment:
use this: https://github.com/openshift/origin/blob/master/pkg/monitortests/testframework/watchnamespaces/monitortest.go#L24

Reply:
I looked at this and I don't think its really applicable at the moment. It relies on a global variable in watchnamespaces to be updated.

I think this would be useful but we would need to move that watchnamespaces utilities somewhere else to make it used from other tests.

I decided to reuse the IsPlatformNamespaces and loop over the namespaces to help at least reduce DRY for platform namespace detection.